packages/edit-site/src/components/add-new-template/add-custom-generic-template-modal.js

```diff
@@ -60,27 +60,22 @@ function AddCustomGenericTemplateModal( {
 		>
 			{ isCreatingTemplate && <TemplateActionsLoadingScreen /> }
 			<form onSubmit={ onCreateTemplate }>
-				<Flex align="flex-start" gap={ 8 }>
-					<FlexItem>
-						<TextControl
-							label={ __( 'Name' ) }
-							value={ title }
-							onChange={ setTitle }
-							placeholder={ defaultTitle }
-							disabled={ isBusy }
-							help={ __(
-								'Describe the template, e.g. "Post with sidebar".'
-							) }
-						/>
-					</FlexItem>
-				</Flex>
-
-				<Flex
-					className="edit-site-custom-generic-template__modal-actions"
-					justify="flex-end"
-					expanded={ false }
-				>
-					<FlexItem>
+				<VStack spacing={ 3 }>
```

Comment:
This is one of many, but we could have continued to Flex and Flex Item but it requires more. Also, before, the first Flex/parent> Flex Item/child wasn't affecting the styles. So I thought it would be cleaner to nest an HStack within a VStack for the same effect but with less.

And it looks like it's been discussed before, to move from Flex to VStack/HStack: https://github.com/WordPress/gutenberg/issues/43085